test_chef_server/kitchen.yml

```diff
@@ -21,7 +25,10 @@ suites:
       - recipe[test_chef_server::default]
       - recipe[test_chef_server::chef_objects]
       - recipe[test_chef_server::chefapi]
+<<<<<<< HEAD
       - recipe[test_chef_server::workstation]
+=======
+>>>>>>> master
```

Comment:
There's some merge artifacts

Reply:
Ugh yeah, all over the place in `examples` and `testapi`